### DISCUSSION TITLE
Is it possible to set this up without building it?

### DISCUSSION THREAD

**Discussion by mazemuster7** [DarkflameUniverse/DarkflameServer#1635] [Q&A] [answered]
I'd rather not have to download multiple other pieces of software just to build something that already has releases.
Is it possible to use one of the releases instead of building it myself?

**Comment by EmosewaMC** [comment]
Yes you can use the releases.  Just skip to past the build step since you will already have the binaries built.  The releases however may be older than the most up to date version of the server, so keep that in mind.

**Comment by EmosewaMC** [comment, marked as answer]
We now have officially supported releases that require little setup and do _not_ require building from source.  Please use those located [here](https://github.com/DarkflameUniverse/DarkflameServer/releases)
